controller/deployments_osioclient_test.go

```diff
@@ -14,6 +15,16 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	deploymentsOsioTestFilePath = "test-files/deployments_osio/"
+)
+
+func readJsonFromTestFile(t *testing.T, testFilePath string) *bytes.Buffer {
```

Comment:
Please use `compareWithGoldenUUIDAgnostic` instead. That function reads and writes a test file (if requested with the `-update` test flag). See below for more on this.